hamster.js

```diff
@@ -2,24 +2,26 @@ var request = require('request');
 var open = require('open');
 
 // 1. Find the correct URL to get adorable hamsters from Giphy!
-request('GIPHY API GET REQUEST URL', function (error, response, body) {
+request('dc6zaTOxFJmzC', function (error, response, body) {
   if (!error && response.statusCode == 200) {
     var responseObject = JSON.parse(body);
     console.log(responseObject);
 
     // 2. Set up an array variable `dataArray` that stores the array of GIF objects. HINT: You'll want to grab the value of the "data" key in the `responseObject` object.
-    // var dataArray = ???;
+    var dataArray = [responseObject];
```

Comment:
if you run the file using `node hamster.js` at this point, you would see in the console one huge object with a key 'data' and it's value is a massive array of more objects. We simply want to set the dataArray variable to that array of objects. `var dataArray = responseObject.data;`